docs/presets/mini.md

```diff
@@ -178,7 +178,11 @@ Prefix for CSS custom properties.
 Utils prefix.
 
 ### preflight
-- **Type:** `boolean`
+- **Type:** `boolean` | `on-demand`
 - **Default:** `true`
 
-Generate preflight.
+Generate preflight css. It can be:
+
+- `true`: always generate preflight.
+- `false`: no preflight.
+- `on-demand`: only generate preflight for used utilities.
```

Comment:
Add`on-demand` mode for `preset-mini` `preflight` option, which can generate the dependency css of theme preflight on demand.